apps/www/components/Nav/index.tsx

```diff
@@ -164,10 +164,10 @@ const Nav = () => {
                     ) : (
                       <>
                         <Button type="default" className="hidden lg:block" asChild>
-                          <Link href="https://supabase.com/dashboard">Sign in</Link>
+                          <Link href="/dashboard">Sign in</Link>
```

Comment:
not 100% tied to this but it's a nice convenience thing so you don't have to manually navigate to log in on the preview origin